src/base58.cpp

```diff
@@ -87,40 +88,37 @@ static const int log256_58Ratio = ceil((std::log(256) / std::log(58)) * baseScal
     return true;
 }
 
+auto Clone(const Span<const unsigned char>& input, int start) -> std::vector<unsigned char> {
```

Comment:
extracted to a method to minimize diff with the next commit where we'll start grouping bytes